### DISCUSSION TITLE
windows下找不到.ddns_go_config.yaml配置文件

### DISCUSSION THREAD

**Discussion by keepzhu** [jeessy2/ddns-go#1171] [Q&A]
已经安装成服务模式-s install，后台运行，但是当前用户文件夹下找不到.ddns_go_config.yaml配置文件（已打开隐藏文件），请问怎么解决？

**Comment by jeessy2** [comment]
网页中点击保存会提示配置文件在哪里

**Reply by keepzhu** [reply]
找到了，第一次运行，要在配置页面填写一下用户密码，点击保存才会生成.ddns_go_config.yaml

**Reply by jeessy2** [reply]
对。没保存是没有的
